packages/muelu/src/Graph/UncoupledAggregation/MueLu_UncoupledAggregationFactory_def.hpp

```diff
@@ -50,6 +50,7 @@
 
 #include <Xpetra_Map.hpp>
 #include <Xpetra_Vector.hpp>
+#include <Xpetra_MultiVector.hpp>
```

Comment:
I think that "Xpetra_Vector.hpp" inludes "Xpetra_MultiVector.hpp" so there is no need for this header